### PR TITLE
Tech Team meetings 06.16 and 06.20

### PR DESCRIPTION
**Pull request by robcraig-LF**
adds two meeting SPDX Tech 06.16 and 06.20

### CODE REVIEW

File: tech/2023-06-20.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## Notes
+
+* Summary of Serialization meeting
+  - essentially two approaches: one more JSON-LD, one more JSON
+  - need a list of problems/concerns/objections for the two approaches
+  - Alexios volunteered to set up list and share link with group for collaboartion on this topic
+
+* Summary of the tech meeting about licensing info
+  - both declaredLlicense and concludedLicense relationships instead of properties
+  - Objections? None expressed
```
Comment (goneall) [inline tech/2023-06-20.md:37]: Suggest adding: `DECISION: use relationships for declaredLicense and concludedLicense` and add to decisions.md file

**Review by goneall — CHANGES_REQUESTED** [review]
One requested change

File: tech/2023-06-20.md (hunk 1 above)
Comment (robcraig-LF) [reply tech/2023-06-20.md:37]: added to line 38. Will at to Decision file on separate PR to avoid conflicts.

**Review by goneall — APPROVED** [review]
LGTM

**Pull request merged**